nixos/modules/programs/foot/default.nix

```diff
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.foot;
+
+  settingsFormat = pkgs.formats.iniWithGlobalSection { listsAsDuplicateKeys = true; };
```

Comment:
Since the global section can also be explicitly named `[main]`, I think it is better to use `pkgs.formats.ini` because it is much more intuitive to set the value of `settings`.

```nix
settings = {
  main = { key1 = "value1"; };
  sectionFoo = { key2 = "value2"; };
};
```

vs

```nix
settings = {
  globalSection = { key1 = "value1"; };
  sections = {
    sectionFoo = { key2 = "value2"; };
  };
};
```

Reply:
Good catch!